api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -295,6 +295,13 @@ type VeleroConfig struct {
 	// Use pod volume file system backup by default for volumes
 	// +optional
 	DefaultVolumesToFSBackup *bool `json:"defaultVolumesToFSBackup,omitempty"`
+	// DisableFsBackup determines whether the NodeAgent should disable file system backup.
+	// When set to true, the NodeAgent runs in non-privileged mode.
+	// Privileged mode is required for backing up block devices.
```

Comment:
Could you clarify this part for me. Block devices are only supported with datamover, not fs-backup. datamover doesn't use the node agent pod for data upload/download, so why does the node agent need to be privileged for this use case?

Reply:
Ah this is from previous implementation where I had one flag for all non-root, so will update.

Reply:
fixed.